service/node.go

```diff
@@ -420,6 +428,47 @@ func (s *service) nodeProbe(ctx context.Context) error {
 	return nil
 }
 
+func (s *service) approveSDC(opts Opts) error {
+
+	for _, systemID := range connectedSystemID {
+		system := s.systems[systemID]
+
+		if system == nil {
+			return nil
```

Comment:
we shouldn't return here, but instead just continue to next systemID. 

Reason being: if there are two connected systems but first one is 00000..., second system should still have SDC approved

Reply:
done